### PR TITLE
IBX-2871: Fixed spacing on some Admin pages

### PR DESCRIPTION
**Pull request by GrabowskiM**
JIRA ref: https://issues.ibexa.co/browse/IBX-2871

### CODE REVIEW

**Review by dew326 — APPROVED** [review]

**Review by lucasOsti — APPROVED** [review]

**Review by alongosz — APPROVED** [review]
@dew326 if you want to merge this one after QA, then you need to take commit message from the PR title, not the default one. If not, lmk once ready to merge, I'll handle this.

**Review by damianz5 — APPROVED** [review]

**Review by tischsoic — APPROVED** [review]

**Review by bogusez — APPROVED** [review]

**Pull request merged**
